haddock-heroku.rb

```diff
@@ -29,3 +29,13 @@
   10.times{ @passwords << Haddock::Password.generate(@length) }
   erb :index
 end
+
+get '/:len'
```

Comment:
I'd prefer if this was under a different 'path' like `/length/:len` so that people wouldn't just be going to `//haddock.mysite/28` but `//haddock.mysite/length/28`